.devcontainer/devcontainer.json

```diff
@@ -8,17 +8,23 @@
   // Features to add to the dev container. More info: https://containers.dev/features.
   "features": {
     "ghcr.io/devcontainers-contrib/features/pnpm": {
-      "version": "9.1.3"
+      "version": "9.8.0"
```

Comment:
We pin the PNPM version to v9.1.3 in our package.json for reproducible builds and lockfile consistency. I've tried using `9` or `9.x.x`, but apparently an exact version is required for the `packageManager` field.

https://github.com/radashi-org/radashi/blob/main/package.json#L15

Reply:
I updated the version more because of the warnings that appeared to me when I was going to install a new dependency. But really, it makes more sense to keep the same as package.json